Build broken on OSX 10.10.5

The build seems to be broken on Mac using JDK 8u73. Following the instructions on `README.md`, I first ran into a build error due to a missing `jMAVlib/src` directory. I attempted to fix that, but then the build fails due to a complaint about missing packages. I'm unfamiliar with the java build process, so cannot troubleshoot further than this. I have included the console output below:

```
Last login: Wed Feb 17 06:19:18 on ttys005
06:19:22 ~/Documents$ git clone https://github.com/DrTon/FlightPlot.git
Cloning into 'FlightPlot'...
remote: Counting objects: 2275, done.
remote: Total 2275 (delta 0), reused 0 (delta 0), pack-reused 2275
Receiving objects: 100% (2275/2275), 3.22 MiB | 1.48 MiB/s, done.
Resolving deltas: 100% (1141/1141), done.
Checking connectivity... done.
06:19:36 ~/Documents$ cd FlightPlot/
06:19:39 (master) ~/Documents/FlightPlot$ ant
Buildfile: /Users/kenz/Documents/FlightPlot/build.xml

make_dirs:
    [mkdir] Created dir: /Users/kenz/Documents/FlightPlot/out/production/FlightPlot

compile:

BUILD FAILED
/Users/kenz/Documents/FlightPlot/build.xml:24: srcdir "/Users/kenz/Documents/FlightPlot/jMAVlib/src" does not exist!

Total time: 0 seconds
```


```
06:19:41 (master) ~/Documents/FlightPlot$ mkdir -p jMAVlib/src
06:19:48 (master) ~/Documents/FlightPlot$ ant
Buildfile: /Users/kenz/Documents/FlightPlot/build.xml

make_dirs:

compile:
   [javac2] Compiling 74 source files to /Users/kenz/Documents/FlightPlot/out/production/FlightPlot
   [javac2] warning: [options] bootstrap class path not set in conjunction with -source 1.6
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/FlightPlot.java:10: error: package me.drton.jmavlib.log does not exist
   [javac2] import me.drton.jmavlib.log.FormatErrorException;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/FlightPlot.java:11: error: package me.drton.jmavlib.log does not exist
   [javac2] import me.drton.jmavlib.log.LogReader;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/FlightPlot.java:12: error: package me.drton.jmavlib.log.px4 does not exist
   [javac2] import me.drton.jmavlib.log.px4.PX4LogReader;
   [javac2]                                ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/FlightPlot.java:13: error: package me.drton.jmavlib.log.ulog does not exist
   [javac2] import me.drton.jmavlib.log.ulog.ULogReader;
   [javac2]                                 ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/FlightPlot.java:100: error: cannot find symbol
   [javac2]     private LogReader logReader = null;
   [javac2]             ^
   [javac2]   symbol:   class LogReader
   [javac2]   location: class FlightPlot
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/LogInfo.java:3: error: package me.drton.jmavlib.log does not exist
   [javac2] import me.drton.jmavlib.log.LogReader;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackExportDialog.java:4: error: package me.drton.jmavlib.log does not exist
   [javac2] import me.drton.jmavlib.log.FormatErrorException;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackExportDialog.java:5: error: package me.drton.jmavlib.log does not exist
   [javac2] import me.drton.jmavlib.log.LogReader;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/FlightPlot.java:1051: error: cannot find symbol
   [javac2]     private void generateSeries() throws IOException, FormatErrorException, InvocationTargetException, NoSuchMethodException, InstantiationException, IllegalAccessException {
   [javac2]                                                       ^
   [javac2]   symbol:   class FormatErrorException
   [javac2]   location: class FlightPlot
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/LogInfo.java:39: error: cannot find symbol
   [javac2]     public void updateInfo(LogReader logReader) {
   [javac2]                            ^
   [javac2]   symbol:   class LogReader
   [javac2]   location: class LogInfo
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackExportDialog.java:50: error: cannot find symbol
   [javac2]     private LogReader logReader;
   [javac2]             ^
   [javac2]   symbol:   class LogReader
   [javac2]   location: class TrackExportDialog
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackExportDialog.java:196: error: cannot find symbol
   [javac2]     public void display(LogReader logReader, Range chartRange) {
   [javac2]                         ^
   [javac2]   symbol:   class LogReader
   [javac2]   location: class TrackExportDialog
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackReader.java:3: error: package me.drton.jmavlib.log does not exist
   [javac2] import me.drton.jmavlib.log.FormatErrorException;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackReader.java:18: error: cannot find symbol
   [javac2]     TrackPoint readNextPoint() throws IOException, FormatErrorException;
   [javac2]                                                    ^
   [javac2]   symbol:   class FormatErrorException
   [javac2]   location: interface TrackReader
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/AbstractTrackReader.java:3: error: package me.drton.jmavlib.log does not exist
   [javac2] import me.drton.jmavlib.log.FormatErrorException;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/AbstractTrackReader.java:4: error: package me.drton.jmavlib.log does not exist
   [javac2] import me.drton.jmavlib.log.LogReader;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/AbstractTrackReader.java:14: error: cannot find symbol
   [javac2]     protected final LogReader reader;
   [javac2]                     ^
   [javac2]   symbol:   class LogReader
   [javac2]   location: class AbstractTrackReader
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/AbstractTrackReader.java:18: error: cannot find symbol
   [javac2]     public AbstractTrackReader(LogReader reader, TrackReaderConfiguration config) throws IOException, FormatErrorException {
   [javac2]                                ^
   [javac2]   symbol:   class LogReader
   [javac2]   location: class AbstractTrackReader
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/AbstractTrackReader.java:18: error: cannot find symbol
   [javac2]     public AbstractTrackReader(LogReader reader, TrackReaderConfiguration config) throws IOException, FormatErrorException {
   [javac2]                                                                                                       ^
   [javac2]   symbol:   class FormatErrorException
   [javac2]   location: class AbstractTrackReader
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/AbstractTrackReader.java:24: error: cannot find symbol
   [javac2]     protected long readUpdate(Map<String, Object> data) throws IOException, FormatErrorException {
   [javac2]                                                                             ^
   [javac2]   symbol:   class FormatErrorException
   [javac2]   location: class AbstractTrackReader
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/PX4TrackReader.java:3: error: package me.drton.jmavlib.log does not exist
   [javac2] import me.drton.jmavlib.log.FormatErrorException;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/PX4TrackReader.java:4: error: package me.drton.jmavlib.log.px4 does not exist
   [javac2] import me.drton.jmavlib.log.px4.PX4LogReader;
   [javac2]                                ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/PX4TrackReader.java:22: error: cannot find symbol
   [javac2]     public PX4TrackReader(PX4LogReader reader, TrackReaderConfiguration config) throws IOException, FormatErrorException {
   [javac2]                           ^
   [javac2]   symbol:   class PX4LogReader
   [javac2]   location: class PX4TrackReader
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/PX4TrackReader.java:22: error: cannot find symbol
   [javac2]     public PX4TrackReader(PX4LogReader reader, TrackReaderConfiguration config) throws IOException, FormatErrorException {
   [javac2]                                                                                                     ^
   [javac2]   symbol:   class FormatErrorException
   [javac2]   location: class PX4TrackReader
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/PX4TrackReader.java:27: error: cannot find symbol
   [javac2]     public TrackPoint readNextPoint() throws IOException, FormatErrorException {
   [javac2]                                                           ^
   [javac2]   symbol:   class FormatErrorException
   [javac2]   location: class PX4TrackReader
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackReaderFactory.java:3: error: package me.drton.jmavlib.log does not exist
   [javac2] import me.drton.jmavlib.log.FormatErrorException;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackReaderFactory.java:4: error: package me.drton.jmavlib.log does not exist
   [javac2] import me.drton.jmavlib.log.LogReader;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackReaderFactory.java:5: error: package me.drton.jmavlib.log.px4 does not exist
   [javac2] import me.drton.jmavlib.log.px4.PX4LogReader;
   [javac2]                                ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackReaderFactory.java:6: error: package me.drton.jmavlib.log.ulog does not exist
   [javac2] import me.drton.jmavlib.log.ulog.ULogReader;
   [javac2]                                 ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackReaderFactory.java:14: error: cannot find symbol
   [javac2]     public static TrackReader getTrackReader(LogReader reader, TrackReaderConfiguration config) throws IOException, FormatErrorException {
   [javac2]                                              ^
   [javac2]   symbol:   class LogReader
   [javac2]   location: class TrackReaderFactory
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackReaderFactory.java:14: error: cannot find symbol
   [javac2]     public static TrackReader getTrackReader(LogReader reader, TrackReaderConfiguration config) throws IOException, FormatErrorException {
   [javac2]                                                                                                                     ^
   [javac2]   symbol:   class FormatErrorException
   [javac2]   location: class TrackReaderFactory
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/ULogTrackReader.java:3: error: package me.drton.jmavlib.log does not exist
   [javac2] import me.drton.jmavlib.log.FormatErrorException;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/ULogTrackReader.java:4: error: package me.drton.jmavlib.log.ulog does not exist
   [javac2] import me.drton.jmavlib.log.ulog.ULogReader;
   [javac2]                                 ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/ULogTrackReader.java:23: error: cannot find symbol
   [javac2]     public ULogTrackReader(ULogReader reader, TrackReaderConfiguration config) throws IOException, FormatErrorException {
   [javac2]                            ^
   [javac2]   symbol:   class ULogReader
   [javac2]   location: class ULogTrackReader
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/ULogTrackReader.java:23: error: cannot find symbol
   [javac2]     public ULogTrackReader(ULogReader reader, TrackReaderConfiguration config) throws IOException, FormatErrorException {
   [javac2]                                                                                                    ^
   [javac2]   symbol:   class FormatErrorException
   [javac2]   location: class ULogTrackReader
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/ULogTrackReader.java:28: error: cannot find symbol
   [javac2]     public TrackPoint readNextPoint() throws IOException, FormatErrorException {
   [javac2]                                                           ^
   [javac2]   symbol:   class FormatErrorException
   [javac2]   location: class ULogTrackReader
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/EulerFromQuaternion.java:3: error: package me.drton.jmavlib.conversion does not exist
   [javac2] import me.drton.jmavlib.conversion.RotationConversion;
   [javac2]                                   ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/GlobalPositionProjection.java:3: error: package me.drton.jmavlib.geo does not exist
   [javac2] import me.drton.jmavlib.geo.GlobalPositionProjector;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/GlobalPositionProjection.java:4: error: package me.drton.jmavlib.geo does not exist
   [javac2] import me.drton.jmavlib.geo.LatLonAlt;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/GlobalPositionProjection.java:13: error: cannot find symbol
   [javac2]     private GlobalPositionProjector positionProjector = new GlobalPositionProjector();
   [javac2]             ^
   [javac2]   symbol:   class GlobalPositionProjector
   [javac2]   location: class GlobalPositionProjection
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/NEDFromBodyProjection.java:4: error: package me.drton.jmavlib.conversion does not exist
   [javac2] import me.drton.jmavlib.conversion.RotationConversion;
   [javac2]                                   ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PosRatePIDControlSimulator.java:5: error: package me.drton.jmavlib.processing does not exist
   [javac2] import me.drton.jmavlib.processing.DelayLine;
   [javac2]                                   ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PosRatePIDControlSimulator.java:20: error: cannot find symbol
   [javac2]     private DelayLine<Double> delayLine = new DelayLine<Double>();
   [javac2]             ^
   [javac2]   symbol:   class DelayLine
   [javac2]   location: class PosRatePIDControlSimulator
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimator.java:3: error: package me.drton.jmavlib.conversion does not exist
   [javac2] import me.drton.jmavlib.conversion.RotationConversion;
   [javac2]                                   ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimator.java:4: error: package me.drton.jmavlib.geo does not exist
   [javac2] import me.drton.jmavlib.geo.GlobalPositionProjector;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimator.java:5: error: package me.drton.jmavlib.geo does not exist
   [javac2] import me.drton.jmavlib.geo.LatLonAlt;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimator.java:6: error: package me.drton.jmavlib.processing does not exist
   [javac2] import me.drton.jmavlib.processing.DelayLine;
   [javac2]                                   ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimator.java:50: error: cannot find symbol
   [javac2]     private GlobalPositionProjector positionProjector;
   [javac2]             ^
   [javac2]   symbol:   class GlobalPositionProjector
   [javac2]   location: class PositionEstimator
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimator.java:52: error: cannot find symbol
   [javac2]     private DelayLine<double[][]> delayLineGPS;
   [javac2]             ^
   [javac2]   symbol:   class DelayLine
   [javac2]   location: class PositionEstimator
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:3: error: package me.drton.jmavlib.conversion does not exist
   [javac2] import me.drton.jmavlib.conversion.RotationConversion;
   [javac2]                                   ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:4: error: package me.drton.jmavlib.geo does not exist
   [javac2] import me.drton.jmavlib.geo.GlobalPositionProjector;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:5: error: package me.drton.jmavlib.geo does not exist
   [javac2] import me.drton.jmavlib.geo.LatLonAlt;
   [javac2]                            ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:6: error: package me.drton.jmavlib.processing does not exist
   [javac2] import me.drton.jmavlib.processing.DelayLine;
   [javac2]                                   ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:72: error: cannot find symbol
   [javac2]     private GlobalPositionProjector positionProjector = new GlobalPositionProjector();
   [javac2]             ^
   [javac2]   symbol:   class GlobalPositionProjector
   [javac2]   location: class PositionEstimatorKF
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:74: error: package DelayLine does not exist
   [javac2]     private List<DelayLine.Tick<GVector>> xBuffer;
   [javac2]                           ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/FlightPlot.java:802: error: cannot find symbol
   [javac2]         LogReader logReaderNew;
   [javac2]         ^
   [javac2]   symbol:   class LogReader
   [javac2]   location: class FlightPlot
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/FlightPlot.java:805: error: cannot find symbol
   [javac2]                 logReaderNew = new PX4LogReader(logFileName);
   [javac2]                                    ^
   [javac2]   symbol:   class PX4LogReader
   [javac2]   location: class FlightPlot
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/FlightPlot.java:807: error: cannot find symbol
   [javac2]                 logReaderNew = new ULogReader(logFileName);
   [javac2]                                    ^
   [javac2]   symbol:   class ULogReader
   [javac2]   location: class FlightPlot
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackReaderFactory.java:15: error: cannot find symbol
   [javac2]         if (reader instanceof PX4LogReader) {
   [javac2]                               ^
   [javac2]   symbol:   class PX4LogReader
   [javac2]   location: class TrackReaderFactory
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackReaderFactory.java:16: error: cannot find symbol
   [javac2]             return new PX4TrackReader((PX4LogReader) reader, config);
   [javac2]                                        ^
   [javac2]   symbol:   class PX4LogReader
   [javac2]   location: class TrackReaderFactory
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackReaderFactory.java:17: error: cannot find symbol
   [javac2]         } else if (reader instanceof ULogReader) {
   [javac2]                                      ^
   [javac2]   symbol:   class ULogReader
   [javac2]   location: class TrackReaderFactory
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/export/TrackReaderFactory.java:18: error: cannot find symbol
   [javac2]             return new ULogTrackReader((ULogReader) reader, config);
   [javac2]                                         ^
   [javac2]   symbol:   class ULogReader
   [javac2]   location: class TrackReaderFactory
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/EulerFromQuaternion.java:55: error: cannot find symbol
   [javac2]         double[] euler = RotationConversion.eulerAnglesByQuaternion(q);
   [javac2]                          ^
   [javac2]   symbol:   variable RotationConversion
   [javac2]   location: class EulerFromQuaternion
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/GlobalPositionProjection.java:13: error: cannot find symbol
   [javac2]     private GlobalPositionProjector positionProjector = new GlobalPositionProjector();
   [javac2]                                                             ^
   [javac2]   symbol:   class GlobalPositionProjector
   [javac2]   location: class GlobalPositionProjection
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/GlobalPositionProjection.java:30: error: cannot find symbol
   [javac2]             positionProjector.init(new LatLonAlt(Double.parseDouble(ref[0]), Double.parseDouble(ref[1]), 0.0));
   [javac2]                                        ^
   [javac2]   symbol:   class LatLonAlt
   [javac2]   location: class GlobalPositionProjection
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/GlobalPositionProjection.java:42: error: cannot find symbol
   [javac2]             LatLonAlt latLonAlt = new LatLonAlt(latNum.doubleValue(), lonNum.doubleValue(), 0.0);
   [javac2]             ^
   [javac2]   symbol:   class LatLonAlt
   [javac2]   location: class GlobalPositionProjection
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/GlobalPositionProjection.java:42: error: cannot find symbol
   [javac2]             LatLonAlt latLonAlt = new LatLonAlt(latNum.doubleValue(), lonNum.doubleValue(), 0.0);
   [javac2]                                       ^
   [javac2]   symbol:   class LatLonAlt
   [javac2]   location: class GlobalPositionProjection
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/NEDFromBodyProjection.java:87: error: cannot find symbol
   [javac2]             r.set(RotationConversion.rotationMatrixByEulerAngles(roll.doubleValue() + param_Att_Offsets[0],
   [javac2]                   ^
   [javac2]   symbol:   variable RotationConversion
   [javac2]   location: class NEDFromBodyProjection
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PosRatePIDControlSimulator.java:20: error: cannot find symbol
   [javac2]     private DelayLine<Double> delayLine = new DelayLine<Double>();
   [javac2]                                               ^
   [javac2]   symbol:   class DelayLine
   [javac2]   location: class PosRatePIDControlSimulator
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimator.java:107: error: cannot find symbol
   [javac2]         delayLineGPS = new DelayLine<double[][]>();
   [javac2]                            ^
   [javac2]   symbol:   class DelayLine
   [javac2]   location: class PositionEstimator
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimator.java:108: error: cannot find symbol
   [javac2]         positionProjector = new GlobalPositionProjector();
   [javac2]                                 ^
   [javac2]   symbol:   class GlobalPositionProjector
   [javac2]   location: class PositionEstimator
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimator.java:162: error: cannot find symbol
   [javac2]             rot.set(RotationConversion.rotationMatrixByEulerAngles(roll.doubleValue(), pitch.doubleValue(),
   [javac2]                     ^
   [javac2]   symbol:   variable RotationConversion
   [javac2]   location: class PositionEstimator
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimator.java:194: error: cannot find symbol
   [javac2]                 positionProjector.init(new LatLonAlt(lat, lon, alt));
   [javac2]                                            ^
   [javac2]   symbol:   class LatLonAlt
   [javac2]   location: class PositionEstimator
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimator.java:199: error: cannot find symbol
   [javac2]                 double[] gpsProj = positionProjector.project(new LatLonAlt(lat, lon, alt));
   [javac2]                                                                  ^
   [javac2]   symbol:   class LatLonAlt
   [javac2]   location: class PositionEstimator
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:72: error: cannot find symbol
   [javac2]     private GlobalPositionProjector positionProjector = new GlobalPositionProjector();
   [javac2]                                                             ^
   [javac2]   symbol:   class GlobalPositionProjector
   [javac2]   location: class PositionEstimatorKF
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:136: error: package DelayLine does not exist
   [javac2]         xBuffer = new ArrayList<DelayLine.Tick<GVector>>();
   [javac2]                                          ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:194: error: cannot find symbol
   [javac2]             rot.set(RotationConversion.rotationMatrixByEulerAngles(roll.doubleValue(), pitch.doubleValue(),
   [javac2]                     ^
   [javac2]   symbol:   variable RotationConversion
   [javac2]   location: class PositionEstimatorKF
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:232: error: cannot find symbol
   [javac2]                 positionProjector.init(new LatLonAlt(lat, lon, alt));
   [javac2]                                            ^
   [javac2]   symbol:   class LatLonAlt
   [javac2]   location: class PositionEstimatorKF
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:236: error: cannot find symbol
   [javac2]                 double[] gpsXYZ = positionProjector.project(new LatLonAlt(lat, lon, alt));
   [javac2]                                                                 ^
   [javac2]   symbol:   class LatLonAlt
   [javac2]   location: class PositionEstimatorKF
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:344: error: package DelayLine does not exist
   [javac2]                 xBuffer.add(new DelayLine.Tick<GVector>(time, new GVector(x)));
   [javac2]                                          ^
   [javac2] /Users/kenz/Documents/FlightPlot/src/me/drton/flightplot/processors/PositionEstimatorKF.java:366: error: package DelayLine does not exist
   [javac2]         DelayLine.Tick<GVector> tick = null;
   [javac2]                  ^
   [javac2] Note: Some input files use unchecked or unsafe operations.
   [javac2] Note: Recompile with -Xlint:unchecked for details.
   [javac2] 81 errors
   [javac2] 1 warning

BUILD FAILED
/Users/kenz/Documents/FlightPlot/build.xml:24: Compile failed; see the compiler error output for details.

Total time: 2 seconds
06:19:51 (master) ~/Documents/FlightPlot$ 

```

--- a/README.md
+++ b/README.md
@@ -27,9 +27,9 @@ Requirements:
  -  Java 6 or newer (JDK, http://www.oracle.com/technetwork/java/javase/downloads/index.html)
  -  ant
 
-Clone repository:
+Clone repository (`--recursive` flag is required to pull in [jMAVlib](https://github.com/DrTon/jMAVlib)):
 ```
-https://github.com/DrTon/FlightPlot.git
+git clone --recursive https://github.com/DrTon/FlightPlot.git
 ```
 
 Build:

